tests/python/driver/tvmc/test_frontends.py

```diff
@@ -367,3 +383,39 @@ def _is_layout_transform(node):
     tvm.relay.analysis.post_order_visit(after["main"], _is_layout_transform)
 
     assert not any(layout_transform_calls), "Unexpected 'layout_transform' call"
+
+
+def test_import_keras_friendly_message(keras_resnet50, monkeypatch):
+    # keras is part of tensorflow
+    monkeypatch.setattr("importlib.import_module", mock_error_on_name("tensorflow"))
+
+    with pytest.raises(TVMCImportError) as e:
+        _ = tvmc.frontends.load_model(keras_resnet50, model_format="keras")
```

Comment:
We should check the error message is correct using the `match` parameter but also the variables aren't necessary:
```suggestion
    with pytest.raises(TVMCImportError, match='Package "keras" is not installed. Hint: "pip install tlcpack[tvmc]"'):
        vmc.frontends.load_model(keras_resnet50, model_format="keras")
```

Reply:
As far as I understand it. The last error will return from main, not from the call to tvmc.frontends.load_model.
isnt match in this case be "keras"?
Can you please review the suggestion again?